### PR TITLE
Add contentSchemas

### PR DESCRIPTION
**Pull request by mariana-mendes**
#### What is the purpose of this pull request?
Add content schema on minicart component.

#### How should this be manually tested?
1.[ Workspace](https://mms--storecomponents.myvtex.com/admin/cms/storefront)
2. `/admin`
3. Header > Minicart

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__mocks__/vtex.native-types/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+export const IOMessage = (props) => <FormattedMessage {...props}/>
```
Comment (vitoria) [inline react/__mocks__/vtex.native-types/index.js]: Não precisa usar o `FormattedMessage`, pode somente retornar o `id`.

File: react/components/MiniCartContent.js [hunk 2]
```diff
@@ -259,17 +266,14 @@ class MiniCartContent extends Component {
     }
 
     if (!orderForm || !itemsToShow.length) {
-      const label =
-        labelMiniCartEmpty || intl.formatMessage({ id: 'store/minicart-empty' })
+      const label = <IOMessage id={labelMiniCartEmpty} />
```
Comment (vitoria) [inline react/components/MiniCartContent.js]: Usa `formatIOMessage` aqui.

File: react/components/MiniCartContent.js [hunk 3]
```diff
@@ -259,17 +266,14 @@ class MiniCartContent extends Component {
...
       return this.renderWithoutItems(label)
     }
 
-    const label =
-      labelButton ||
-      intl.formatMessage({ id: 'store/finish-shopping-button-label' })
-    const labelDiscount = intl.formatMessage({
-      id: 'store/minicart-content-footer-discount',
-    })
+    const label = <IOMessage id={labelButton} />
```
Comment (vitoria) [inline react/components/MiniCartContent.js]: Same here.

File: react/components/MiniCartContent.js [hunk 4]
```diff
@@ -259,17 +266,14 @@ class MiniCartContent extends Component {
...
+    const labelDiscount = (
+      <IOMessage id={'store/minicart-content-footer-discount'} />
```
Comment (vitoria) [inline react/components/MiniCartContent.js]: Same here.

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (vitoria) [inline yarn.lock]: Why did you add it?

**Comment by Klynger** [comment]
Going to close this PR due to inactivity
